src/components/Panel/Panel.scss

```diff
@@ -326,3 +326,9 @@ $CommandBarHeight: 40px;
     margin-top: 30px;
   }
 }
+
+.ms-Panel-screenReader {
```

Comment:
Yes, the `.ms-u-screenReaderOnly` utility class will visually hide content that should still be read by screen readers.